Header header
int32 Tag_id
int32 quality #0:no solution 1:bad quality 2:good quality

# Position Data
float64 x
float64 y
float64 z

# Accuracy Statistics
float32 x_sigma
float32 y_sigma
float32 z_sigma
